src/app/tests/suites/certification/Test_TC_S_2_6.yaml

```diff
@@ -28,10 +28,9 @@ config:
 tests:
     - label: "Precondition: Commission DUT to TH1"
       verification: |
-          Once DUT reach the  commissionable state send the following command on TH1:
-          pairing onnetwork 1 20202021
-          Verify the commissioning completed with success on TH(chip-tool) from DUT
-          [1650455358.501816][4366:4371] CHIP:TOO: Device commissioning completed with success
+          - DUT should be commissioned onto TH1, TH2, and TH3.
+          - The Scene Table capacity for a given fabric SHALL be less than half (rounded down towards 0) of the Scene Table entries (as indicated in the SceneTableSize attribute).
+          - MaxRemainingCapacity is (SceneTableSize-1)/2.
```

Comment:
The Precondition from previous must also be present along with new precondition. please refer the VS document.

Reply:
Added the TH1 commissioning back.